docs/sources/datasources/aws-cloudwatch/_index.md

```diff
@@ -22,6 +22,12 @@ For authentication options and configuration details, see [AWS authentication]({
 
 ### CloudWatch specific data source configuration
 
+### CloudWatch Logs
```

Comment:
Think this header should be moved so that it's displayed under `##Namespaces of Custom Metrics`, since that option is displayed higher up in the config editor. Also, don't think there should be a table layout to describe this since we don't use it for any other field.